jvb/src/main/kotlin/org/jitsi/videobridge/load_management/StealDetection.kt

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright @ 2025 - present 8x8, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.videobridge.load_management
+
+import org.jitsi.metrics.DoubleGaugeMetric
+import org.jitsi.videobridge.metrics.VideobridgeMetricsContainer.Companion.instance as metricsContainer
+
+object StealDetection {
```

Comment:
It'd be nice to have unit tests for this.  Probably by making it a class rather than an object, and having some way to feed it dummy contents of /proc/stat.